.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
```

Comment:
You may want to add a few more things to this: See https://github.com/connexta/discovery-api/blob/master/.gitignore